Microsoft.Web.Graph.WebRole/Controllers/HomeController.cs

```diff
@@ -3,14 +3,17 @@
 using System.Linq;
 using System.Web;
 using System.Web.Mvc;
+using Microsoft.Web.Graph.WebRole.ViewModels;
 
 namespace Microsoft.Web.Graph.WebRole.Controllers
 {
     public class HomeController : BaseController
     {
         public ActionResult Index()
         {
-            return View();
+            ViewModelBase model = new ViewModelBase(HttpContext.ApplicationInstance.Context);
+            model.PageTitle = "Microsoft.Web.Graph.IO Prototype";
```

Comment:
What about just "Microsoft Graph"? I can think of a couple people who would have a problem with seeing "prototype" being displayed on the page :)